### PR TITLE
fselect: add page

### PR DESCRIPTION
**Pull request by derphilipp**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).

### CODE REVIEW

File: pages/common/fselect.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# fselect
+
+> Find files with SQL-like queries.
+
+- Find temporary or config files (full path and size):
```
Comment (agnivade) [inline pages/common/fselect.md]: How about - "Select full path and size from temporary or config files in a given directory."

Comment (derphilipp) [reply pages/common/fselect.md]: Sure, why not :)

File: pages/common/fselect.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# fselect
+
+> Find files with SQL-like queries.
+
+- Select full path and size from temporary or config files in a given directory:
+
+`fselect size, path from /home/user where name = '*.cfg' or name = '*.tmp'`
```
Comment (mebeim) [inline pages/common/fselect.md]:
```suggestion
`fselect size, path from {{path/to/directory}} where name = {{'*.cfg'}} or name = {{'*.tmp'}}`
```

File: pages/common/fselect.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- Find square images:
+
+`fselect path from /home/user/Photos where width = height`
```
Comment (mebeim) [inline pages/common/fselect.md]:
```suggestion
`fselect path from {{path/to/directory}} where width = height`
```

File: pages/common/fselect.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+- Find old-school rap MP3 files:
+
+`fselect path from /home/user/music where genre = Rap and bitrate = 320 and mp3_year lt 2000`
```
Comment (mebeim) [inline pages/common/fselect.md]:
```suggestion
`fselect path from {{path/to/directory}} where genre = {{Rap}} and bitrate = {{320}} and mp3_year lt {{2000}}`
```

File: pages/common/fselect.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+- Select full path and size from temporary or config files in a given directory:
+
+`fselect size, path from /home/user where name = '*.cfg' or name = '*.tmp'`
+
+- Find square images:
+
+`fselect path from /home/user/Photos where width = height`
+
+- Find old-school rap MP3 files:
```
Comment (mebeim) [inline pages/common/fselect.md]:
```suggestion
- Find old-school rap 320kbps MP3 files:
```

File: pages/common/fselect.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+`fselect path from /home/user/music where genre = Rap and bitrate = 320 and mp3_year lt 2000`
+
+- Output as json:
+
+`fselect size, path from /home/user limit 5 into json`
```
Comment (mebeim) [inline pages/common/fselect.md]:
```suggestion
`fselect size, path from {{path/to/directory}} limit {{5}} into json`
```

File: pages/common/fselect.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+- Find square images:
+
+`fselect path from /home/user/Photos where width = height`
+
+- Find old-school rap MP3 files:
+
+`fselect path from /home/user/music where genre = Rap and bitrate = 320 and mp3_year lt 2000`
+
+- Output as json:
```
Comment (mebeim) [inline pages/common/fselect.md]:
```suggestion
- Select only the first 5 results and output as JSON:
```

File: pages/common/fselect.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+
+`fselect size, path from /home/user limit 5 into json`
+
+- Aggregate functions:
+
+`fselect "MIN(size), MAX(size), AVG(size), SUM(size), COUNT(*) from /home/user/Downloads"`
```
Comment (mebeim) [inline pages/common/fselect.md]:
```suggestion
`fselect "{{MIN(size), MAX(size), AVG(size), SUM(size), COUNT(*)}} from {{path/to/directory}}"`
```

File: pages/common/fselect.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+
+- Find old-school rap MP3 files:
+
+`fselect path from /home/user/music where genre = Rap and bitrate = 320 and mp3_year lt 2000`
+
+- Output as json:
+
+`fselect size, path from /home/user limit 5 into json`
+
+- Aggregate functions:
```
Comment (mebeim) [inline pages/common/fselect.md]:
```suggestion
- Use SQL aggregate functions to calculate minimum, maximum and average size of files in a directory:
```

File: pages/common/fselect.md [hunk 10]
```diff
@@ -0,0 +1,23 @@
+# fselect
+
+> Find files with SQL-like queries.
```
Comment (mebeim) [inline pages/common/fselect.md]:
```suggestion
> Find files with SQL-like queries.
> Homepage: <https://github.com/jhspetersson/fselect>.
```

**Comment by mebeim** [comment]
Hello @derphilipp could you please commit the remaining changes so we can finally merge this? It's been a while, and I don't quite understand why you only applied half the suggested changes.

File: pages/common/fselect.md [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+> Homepage: <https://github.com/jhspetersson/fselect>.
```
Comment (mebeim) [inline pages/common/fselect.md]:
```suggestion
> More information: <https://github.com/jhspetersson/fselect>.
```
I missed this, sorry. Looks good after this.

**Pull request merged**